extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
+    const data = {};
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
+    for (let i = 0; i < inputs.length; i++) {
+      const input = inputs[i];
+      if (!input.name || isDisabled_(input, /*checkAncestors*/ true)) {
+        continue;
+      }
+
+      if (input.type == 'checkbox' || input.type == 'radio') {
```

Comment:
Ah. Good point. I need to update this to add it to an array, and need to update `serializeQueryString` to accept arrays as value and serialize it properly.

The end result should be something like `checkbox_group1=value1&checkbox_group1=value2` ...